src/sentry/runner/initializer.py

```diff
@@ -400,12 +400,7 @@ def validate_options(settings):
 def __model_unpickle_compat(model_id, attrs=None, factory=None):
     from django import VERSION
 
-    if VERSION[:2] == (1, 9):
-        attrs = [] if attrs is None else attrs
-        factory = django.db.models.base.simple_class_factory if factory is None else factory
-        return model_unpickle(model_id, attrs, factory)
-    # TODO(joshuarli): unverified on 1.11, but i'm doing this to unblock tests for now
-    elif VERSION[:2] in [(1, 10), (1, 11)]:
+    if VERSION[:2] in [(1, 10), (1, 11)]:
         return model_unpickle(model_id)
     else:
         raise NotImplementedError
```

Comment:
We could actually just remove this entirely now, and just leave the `__simple_class_factory_compat` stuff in place.

Reply:
I was thinking about that, but felt like we should raise NotImplementedError as a reminder for whenever we upgrade to Django 2.